extensions/amp-access/amp-access.md

```diff
@@ -25,7 +30,7 @@ limitations under the License.
   </tr>
   <tr>
     <td class="col-fourty"><strong>Availability</strong></td>
-    <td>Stable</td>
+    <td>In development</td>
```

Comment:
Okay.  If it's stable:
@dvoytenko 
- Should I also remove the callout at the top as well? (...This extension is under active development, and the version number of the specification section should provide guidance to its evolution.)  
- What should be done about the section on "Specification v0.5"?